integration/benchmark/account/account.go

```diff
@@ -0,0 +1,87 @@
+package account
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+
+	flowsdk "github.com/onflow/flow-go-sdk"
+
+	"github.com/onflow/flow-go-sdk/access"
+	"github.com/onflow/flow-go-sdk/crypto"
+)
+
+type FlowAccount struct {
+	Address *flowsdk.Address
+	ID      int
+
+	keys   *keystore
+	signer crypto.InMemorySigner
+}
+
+func New(i int, address *flowsdk.Address, accountKeys []*flowsdk.AccountKey, signer crypto.InMemorySigner) *FlowAccount {
+	keys := make([]*accountKey, 0, len(accountKeys))
+	for _, key := range accountKeys {
+		keys = append(keys, &accountKey{
+			AccountKey: key,
+			Address:    address,
+			Signer:     signer,
```

Comment:
given signer is not concurrency safe, is it safe to reuse the same object across keys?

Reply:
you were very right here-- it was useless to lock the key, given that we were reusing the signer across keys.